project-1_test/test_main.py

```diff
@@ -0,0 +1,81 @@
+from main import *
+import pytest
+
+
+# Test cases for open spreadsheet function
+def test_open_one():
```

Comment:
your test case name isn't descriptive enough. `_open_one` what? I'd recommend :

1. Merge all the `_open_[number]` cases into as `test_spreadsheet_connection` and add the assertions there

Reply:
I have done that.